app/cypress/support/commands.js

```diff
@@ -524,46 +524,46 @@ Cypress.Commands.add(
 Cypress.Commands.add(
   "checkFundingAgreementForm",
```

Comment:
I got confused about how to use this function, so refactored a little to put the arguments and checks in the same order, and removed default values so you can't accidentally mix up their order